src/components/projects/projectId/heuristics/HeuristicsHeader.tsx

```diff
@@ -160,84 +163,101 @@ export default function HeuristicsHeader(props: HeuristicsHeaderProps) {
-                                    Weak supervision
-                                </button>
-                            </Tooltip>
-                        ) : (<Tooltip content={TOOLTIPS_DICT.HEURISTICS.SELECT_AT_LEAST_ONE_VALID_HEURISTIC} color="invert" placement="top">
-                            <button className="bg-indigo-700 text-white text-xs font-semibold mr-3 px-4 py-2 rounded-md border opacity-50 cursor-not-allowed" disabled={true}>
-                                Weak supervision
-                            </button>
-                        </Tooltip>)}
-                    </>) : (<Tooltip content={TOOLTIPS_DICT.HEURISTICS.SELECT_AT_LEAST_ONE_HEURISTIC} color="invert" placement="top">
-                        <button className="bg-indigo-700 text-white text-xs font-semibold mr-3 px-4 py-2 rounded-md border opacity-50 cursor-not-allowed" disabled={true}>
-                            Weak supervision
-                        </button>
-                    </Tooltip>)}
+                            <KernButton
+                                text="Weak supervision"
+                                buttonColor="indigo"
+                                solidTheme={true}
+                                textColor="white"
+                                disabled={props.tokenizationProgress < 1}
+                                tooltip={TOOLTIPS_DICT.HEURISTICS.WEAK_SUPERVISION}
+                                tooltipPlacement="top"
+                                size="small"
+                                className="mr-3"
+                                onClick={startWeakSupervision}
+                            />
+                        ) : (
+                            <KernButton
+                                text="Weak supervision"
+                                buttonColor="indigo"
+                                solidTheme={true}
+                                textColor="white"
+                                disabled={true}
+                                tooltip={TOOLTIPS_DICT.HEURISTICS.SELECT_AT_LEAST_ONE_VALID_HEURISTIC}
+                                tooltipPlacement="top"
+                                size="small"
+                                className="mr-3"
+                            />
+                        )}
+                    </>) : (
+                        <KernButton
+                            text="Weak supervision"
+                            buttonColor="indigo"
+                            solidTheme={true}
+                            textColor="white"
+                            disabled={true}
+                            tooltip={TOOLTIPS_DICT.HEURISTICS.SELECT_AT_LEAST_ONE_HEURISTIC}
+                            tooltipPlacement="top"
+                            size="small"
+                            className="mr-3"
+                        />
```

Comment:
only tooltip & disabled seems to change between. could be combined into one button